doc/integrator/objectstorage.rst

```diff
@@ -177,6 +177,13 @@ Azure:
 
 `Some more information <https://github.com/mapserver/mapserver/wiki/Render-images-straight-out-of-S3-with-the-vsicurl-driver>`_
 
+.. note::
+
+   If you want to be able to use different Bucket or Container in different environment
```

Comment:
Suggestion:

If you want to use different buckets or containers in different environments (such as integration / production)